WORKING_GROUPS.md

```diff
@@ -76,6 +76,105 @@ Its responsibilities are:
 The current members can be found in their
 [README](https://github.com/iojs/build#people).
 
+### Tracing
+
+The tracing working group's purpose is to increase the
+transparency of software written in io.js.
+
+Its responsibilities are:
+* Collaboration with V8 to integrate with `trace_event`.
+* Maintinence and iteration on AsyncWrap.
+* Maintinence and improvements to system tracing support (DTrace, LTTng, etc).
+* Documention of tracing and debugging techniques.
+* Fostering a tracing and debugging ecosystem.
+
+The current members can be found in their
+[README](https://github.com/iojs/tracing-wg#people).
+
+### i18n
+
+The i18n working groups handle more than just translations. They
+are endpoints for community members to collaborate with each
+other in their language of choice.
+
+Each team is organized around a common spoken language. Each
+language community might then produce multiple localizations for
+various project resources.
+
+Their responsibilities are:
+* Translationg of any io.js materials they believe are relevant to their
+community.
+* Review processes for keeping translations up
+to date and of high quality.
+* Social media channels in their language.
+* Promotion of io.js speakers for meetups and conferences in their
+language.
+
+Membership is maintained by each language community.
+
+* [iojs-bn](http://github.com/iojs/iojs-bn)
```

Comment:
we have 2 or 3 more already, but it will slow down naturally soon enough. i don't see how a table would work